code/modules/clothing/masks/hailer.dm

```diff
@@ -39,28 +41,161 @@
 	if(istype(W, /obj/item/weapon/screwdriver))
 		switch(aggressiveness)
 			if(1)
-				user << "<span class='notice'>You set the restrictor to the middle position.</span>"
+				user << "<span class='notice'>You set the aggressiveness restrictor to the second position.</span>"
```

Comment:
Making it just Level X would save the whole switch. Alternatively moving ordinal numbers generation to some helper, could be useful elsewhere.